can you please also add : ["poolformer", "dpt", "roformer", "mpnet", "deberta", "deberta_v2", "big_bird"]
```
2025-04-07T11:54:59.3014635Z =========================== short test summary info ============================
2025-04-07T11:54:59.3015409Z FAILED tests/onnxruntime/test_modeling.py::ORTModelForMaskedLMIntegrationTest::test_compare_to_transformers_02_big_bird - AssertionError: Tensor-likes are not close!
2025-04-07T11:54:59.3015420Z 
2025-04-07T11:54:59.3015532Z Mismatched elements: 36 / 453222 (0.0%)
2025-04-07T11:54:59.3015796Z Greatest absolute difference: 1.6531025742728464e+31 at index (0, 0, 2) (up to 0.0001 allowed)
2025-04-07T11:54:59.3016081Z Greatest relative difference: 2693914.25 at index (0, 0, 0) (up to 0.0001 allowed)
2025-04-07T11:54:59.3016599Z FAILED tests/onnxruntime/test_modeling.py::ORTModelForMaskedLMIntegrationTest::test_compare_to_transformers_06_deberta - AssertionError: Tensor-likes are not close!
2025-04-07T11:54:59.3016607Z 
2025-04-07T11:54:59.3016715Z Mismatched elements: 15360 / 15360 (100.0%)
2025-04-07T11:54:59.3016973Z Greatest absolute difference: 2.5342160989229478e+26 at index (0, 0, 463) (up to 0.0001 allowed)
2025-04-07T11:54:59.3017208Z Greatest relative difference: 9.20669464448467e+19 at index (0, 4, 110) (up to 0.0001 allowed)
2025-04-07T11:54:59.3017783Z FAILED tests/onnxruntime/test_modeling.py::ORTModelForMaskedLMIntegrationTest::test_compare_to_transformers_07_deberta_v2 - AssertionError: Tensor-likes are not close!
2025-04-07T11:54:59.3017789Z 
2025-04-07T11:54:59.3017904Z Mismatched elements: 1152007 / 1152009 (100.0%)
2025-04-07T11:54:59.3018102Z Greatest absolute difference: nan at index (0, 0, 65) (up to 0.0001 allowed)
2025-04-07T11:54:59.3018295Z Greatest relative difference: nan at index (0, 0, 65) (up to 0.0001 allowed)
2025-04-07T11:54:59.3018813Z FAILED tests/onnxruntime/test_modeling.py::ORTModelForMaskedLMIntegrationTest::test_compare_to_transformers_12_mobilebert - AssertionError: Tensor-likes are not close!
2025-04-07T11:54:59.3018822Z 
2025-04-07T11:54:59.3018927Z Mismatched elements: 15168 / 26976 (56.2%)
2025-04-07T11:54:59.3019123Z Greatest absolute difference: nan at index (0, 0, 2) (up to 0.0001 allowed)
2025-04-07T11:54:59.3019309Z Greatest relative difference: nan at index (0, 0, 2) (up to 0.0001 allowed)
2025-04-07T11:54:59.3019814Z FAILED tests/onnxruntime/test_modeling.py::ORTModelForMaskedLMIntegrationTest::test_compare_to_transformers_13_mpnet - AssertionError: Tensor-likes are not close!
2025-04-07T11:54:59.3019819Z 
2025-04-07T11:54:59.3019921Z Mismatched elements: 26831 / 27000 (99.4%)
2025-04-07T11:54:59.3020175Z Greatest absolute difference: 1.6455917599407446e+31 at index (0, 0, 466) (up to 0.0001 allowed)
2025-04-07T11:54:59.3020395Z Greatest relative difference: 22199113728.0 at index (0, 5, 2) (up to 0.0001 allowed)
2025-04-07T11:54:59.3020908Z FAILED tests/onnxruntime/test_modeling.py::ORTModelForMaskedLMIntegrationTest::test_compare_to_transformers_16_roformer - AssertionError: Tensor-likes are not close!
2025-04-07T11:54:59.3020973Z 
2025-04-07T11:54:59.3021073Z Mismatched elements: 9 / 450000 (0.0%)
2025-04-07T11:54:59.3021296Z Greatest absolute difference: 523218419712.0 at index (0, 0, 0) (up to 0.0001 allowed)
2025-04-07T11:54:59.3021505Z Greatest relative difference: 323862.34375 at index (0, 0, 0) (up to 0.0001 allowed)
2025-04-07T11:54:59.3022028Z FAILED tests/onnxruntime/test_modeling.py::ORTModelForMaskedLMIntegrationTest::test_compare_to_transformers_17_squeezebert - AssertionError: Tensor-likes are not close!
2025-04-07T11:54:59.3022033Z 
2025-04-07T11:54:59.3022136Z Mismatched elements: 26976 / 26976 (100.0%)
2025-04-07T11:54:59.3022330Z Greatest absolute difference: nan at index (0, 0, 0) (up to 0.0001 allowed)
2025-04-07T11:54:59.3022516Z Greatest relative difference: nan at index (0, 0, 0) (up to 0.0001 allowed)
2025-04-07T11:54:59.3023102Z FAILED tests/onnxruntime/test_modeling.py::ORTModelForMaskedLMIntegrationTest::test_pipeline_ort_model_12_mobilebert - AssertionError: nan not greater than or equal to 0.0
2025-04-07T11:54:59.3023718Z FAILED tests/onnxruntime/test_modeling.py::ORTModelForImageClassificationIntegrationTest::test_pipeline_ort_model_11_poolformer - AssertionError: nan not greater than or equal to 0.0
2025-04-07T11:54:59.3024263Z FAILED tests/onnxruntime/test_modeling.py::ORTModelForSemanticSegmentationIntegrationTest::test_compare_to_transformers_1_dpt - AssertionError: Tensor-likes are not close!
2025-04-07T11:54:59.3024268Z 
2025-04-07T11:54:59.3024369Z Mismatched elements: 2048 / 2048 (100.0%)
2025-04-07T11:54:59.3024639Z Greatest absolute difference: 1.6033836527025424e+34 at index (0, 0, 10, 20) (up to 0.0001 allowed)
2025-04-07T11:54:59.3024924Z Greatest relative difference: 1.0000416040420532 at index (0, 1, 10, 20) (up to 0.0001 allowed)
2025-04-07T11:54:59.3025590Z FAILED tests/onnxruntime/test_modeling.py::ORTModelForImageClassificationIntegrationTest::test_compare_to_transformers_11_poolformer - AssertionError: Tensor-likes are not close!
2025-04-07T11:54:59.3025605Z 
2025-04-07T11:54:59.3025703Z Mismatched elements: 2 / 2 (100.0%)
2025-04-07T11:54:59.3025924Z Greatest absolute difference: 8058407156187136.0 at index (0, 1) (up to 0.0001 allowed)
2025-04-07T11:54:59.3026132Z Greatest relative difference: 1692201984.0 at index (0, 1) (up to 0.0001 allowed)
```

--- a/src/transformers/models/big_bird/modeling_big_bird.py
+++ b/src/transformers/models/big_bird/modeling_big_bird.py
@@ -1769,6 +1769,8 @@ class BigBirdPreTrainedModel(PreTrainedModel):
         elif isinstance(module, nn.LayerNorm):
             module.bias.data.zero_()
             module.weight.data.fill_(1.0)
+        elif isinstance(module, BigBirdLMPredictionHead):
+            module.bias.data.zero_()
 
 
 BIG_BIRD_START_DOCSTRING = r"""

--- a/src/transformers/models/deberta/modeling_deberta.py
+++ b/src/transformers/models/deberta/modeling_deberta.py
@@ -539,11 +539,11 @@ class DebertaLayer(nn.Module):
             return (layer_output, None)
 
 
-class DebertaEncoder(PreTrainedModel):
+class DebertaEncoder(nn.Module):
     """Modified BertEncoder with relative position bias support"""
 
     def __init__(self, config):
-        super().__init__(config)
+        super().__init__()
         self.layer = nn.ModuleList([DebertaLayer(config) for _ in range(config.num_hidden_layers)])
         self.relative_attention = getattr(config, "relative_attention", False)
         if self.relative_attention:
@@ -655,6 +655,14 @@ class DebertaPreTrainedModel(PreTrainedModel):
             module.weight.data.normal_(mean=0.0, std=self.config.initializer_range)
             if module.padding_idx is not None:
                 module.weight.data[module.padding_idx].zero_()
+        elif isinstance(module, (nn.LayerNorm, DebertaLayerNorm)):
+            module.weight.data.fill_(1.0)
+            module.bias.data.zero_()
+        elif isinstance(module, DisentangledSelfAttention):
+            module.q_bias.data.zero_()
+            module.v_bias.data.zero_()
+        elif isinstance(module, (LegacyDebertaLMPredictionHead, DebertaLMPredictionHead)):
+            module.bias.data.zero_()
 
 
 DEBERTA_START_DOCSTRING = r"""

--- a/src/transformers/models/deberta_v2/modeling_deberta_v2.py
+++ b/src/transformers/models/deberta_v2/modeling_deberta_v2.py
@@ -703,7 +703,6 @@ class DebertaV2Encoder(nn.Module):
         )
 
 
-# Copied from transformers.models.deberta.modeling_deberta.DebertaPreTrainedModel with Deberta->DebertaV2
 class DebertaV2PreTrainedModel(PreTrainedModel):
     """
     An abstract class to handle weights initialization and a simple interface for downloading and loading pretrained
@@ -727,6 +726,11 @@ class DebertaV2PreTrainedModel(PreTrainedModel):
             module.weight.data.normal_(mean=0.0, std=self.config.initializer_range)
             if module.padding_idx is not None:
                 module.weight.data[module.padding_idx].zero_()
+        elif isinstance(module, nn.LayerNorm):
+            module.weight.data.fill_(1.0)
+            module.bias.data.zero_()
+        elif isinstance(module, (LegacyDebertaV2LMPredictionHead, DebertaV2LMPredictionHead)):
+            module.bias.data.zero_()
 
 
 DEBERTA_START_DOCSTRING = r"""

--- a/src/transformers/models/mpnet/modeling_mpnet.py
+++ b/src/transformers/models/mpnet/modeling_mpnet.py
@@ -63,6 +63,8 @@ class MPNetPreTrainedModel(PreTrainedModel):
         elif isinstance(module, nn.LayerNorm):
             module.bias.data.zero_()
             module.weight.data.fill_(1.0)
+        elif isinstance(module, MPNetLMHead):
+            module.bias.data.zero_()
 
 
 class MPNetEmbeddings(nn.Module):

--- a/src/transformers/models/poolformer/modeling_poolformer.py
+++ b/src/transformers/models/poolformer/modeling_poolformer.py
@@ -272,9 +272,13 @@ class PoolFormerPreTrainedModel(PreTrainedModel):
             module.weight.data.normal_(mean=0.0, std=self.config.initializer_range)
             if module.bias is not None:
                 module.bias.data.zero_()
-        elif isinstance(module, nn.LayerNorm):
+        elif isinstance(module, nn.GroupNorm):
             module.bias.data.zero_()
             module.weight.data.fill_(1.0)
+        elif isinstance(module, PoolFormerLayer):
+            if hasattr(module, "layer_scale_1"):
+                module.layer_scale_1.data.fill_(self.config.layer_scale_init_value)
+                module.layer_scale_2.data.fill_(self.config.layer_scale_init_value)
 
 
 POOLFORMER_START_DOCSTRING = r"""

--- a/src/transformers/models/roformer/modeling_roformer.py
+++ b/src/transformers/models/roformer/modeling_roformer.py
@@ -699,6 +699,8 @@ class RoFormerPreTrainedModel(PreTrainedModel):
         elif isinstance(module, nn.LayerNorm):
             module.bias.data.zero_()
             module.weight.data.fill_(1.0)
+        elif isinstance(module, RoFormerLMPredictionHead):
+            module.bias.data.zero_()
 
 
 ROFORMER_START_DOCSTRING = r"""

